Cargo.toml

```diff
@@ -37,11 +37,11 @@ sha1 = { version = "0.10", default-features = false }
 sha2 = "0.10"
 num-derive = "0.4"
 num-traits = { version = "0.2", default-features = false }
-picky = "7.0.0-rc.11"
-picky-asn1 = "0.10"
-picky-asn1-der = "0.5"
-picky-asn1-x509 = "0.14"
-picky-krb = "0.9"
+picky = { path = "../picky-rs/picky" }
+picky-asn1 = { path = "../picky-rs/picky-asn1" }
+picky-asn1-der = { path = "../picky-rs/picky-asn1-der" }
+picky-asn1-x509 = { path = "../picky-rs/picky-asn1-x509" }
+picky-krb = { path = "../picky-rs/picky-krb" }
```

Comment:
The PR depends on changes in picky-rs: https://github.com/Devolutions/picky-rs/pull/342

Reply:
https://github.com/Devolutions/picky-rs/pull/343

Reply:
done